src/yacfg/yacfg.py

```diff
@@ -136,6 +136,11 @@ def override_value_map_keys(value: Dict[str, Any]) -> Dict[str, Any]:
         """
         return {override_value(key, key): val for key, val in value.items()}
 
+    def zip_lists(value: List[Any], pairing: List[Any]) -> List[(Any, Any)]:
+        if not isinstance(value, List) or not isinstance(pairing, List):
+            return []
+        return zip(value, pairing)
```

Comment:
Would be nice to have a test/usage example for this. Having the actual template this will operate on only in a separate repo sucks.

Reply:
I agree and will try to add the test into profile_test.sh.